impl/src/main/java/com/okta/sdk/impl/ds/DefaultResourceConverter.java

```diff
@@ -74,16 +73,7 @@ private Object toMapValue(final AbstractResource resource, final String propName
         if (value instanceof Map) {
             //Since defaultModel is a map, the DataStore thinks it is a Resource. This causes the code to crash later one as Resources
             //do need to have an href property
-
-            //if the property is a reference, don't write the entire object - just the href will do:
-            //TODO need to change this to write the entire object because this code defeats the purpose of entity expansion
-            //     when this code gets called (returning the reference instead of the whole object that is returned from Okta)
-            if(Strings.hasText(resource.getResourceHref())) {
-                return this.referenceFactory.createReference(propName, (Map) value);
-            }
-            else{
-                return this.referenceFactory.createUnmaterializedReference(propName, (Map) value);
-            }
+            return this.referenceFactory.createUnmaterializedReference(propName, (Map) value);
```

Comment:
This was a left over from the Stormpath SDK, with the Okta data model, everything is included directly in the model